lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -36,8 +36,10 @@ class Map
 module Fluent
   # fluentd output plugin for the Stackdriver Logging API
   class GoogleCloudOutput < BufferedOutput
-    # Constants for service names and resource types.
+    # Constants used by the plugin and its tests.
```

Comment:
s/used/shared/? But I fail to see the reason behind this change...
How about renaming this module to `ServiceConstants` and leaving the comment as-is?